Library/Formula/double-conversion.rb

```diff
@@ -0,0 +1,16 @@
+class DoubleConversion < Formula
+  homepage "https://github.com/floitsch/double-conversion"
+  url "https://github.com/floitsch/double-conversion/archive/v1.1.5.tar.gz"
+  sha256 "03b976675171923a726d100f21a9b85c1c33e06578568fbc92b13be96147d932"
+
+  head "https://github.com/floitsch/double-conversion.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    mkdir "dc-build" do
+      system "cmake", "..", *std_cmake_args
+      system "make", "install"
+    end
+  end
+end
```

Comment:
Can a test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!